src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderCourseLocationController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.RoatpCourseManagement.Application.Standards.Queries.GetProviderCourseLocation;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RoatpCourseManagement.Api.Controllers
+{
+    [ApiController]
+    public class ProviderCourseLocationController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<ProviderCourseLocationController> _logger;
+
+        public ProviderCourseLocationController(ILogger<ProviderCourseLocationController> logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [HttpGet]
+        [Route("providers/{ukprn}/courses/{larsCode}/provider-course-locations")]
```

Comment:
There is an inconsistency between this and work done in CSP-114. You have a controller `ProviderCourseLocationRegionsController` in there and the endpoint has `standardsubregions` on it. 
I think we should have only one Controller `ProviderCourseLocationsController` and this should have only one endpoint that looks like this... `providers/{ukprn}/courses/{larsCode}/locations?{locationtype}`.

Reply:
ok I see, the `standardssubregions` is doing a little bit more than just getting locations for the course. However the endpoint names are inconsistent. This one needs to be updated to `providers/{ukprn}/courses/{larsCode}/locations/provider-locations`.